recipes/DataProfiler/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set name = "DataProfiler" %}
+{% set version = "0.8.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DataProfiler-{{ version }}.tar.gz
+  sha256: f544b0ab58679ac4c93155439f6fd817241fb480290284a2dde8ce5b6b9e7846
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+    - h5py >=2.10.0
+    - wheel >=0.33.1
+    - future >=0.18.2
+    - numpy >=1.21.1
+    - pandas >=1.1.2
+    - python-dateutil >=2.7.5
+    - pytz >=2020.1
+    - six >=1.15.0
+    - pyarrow >=1.0.1
+    - chardet >=3.0.4
+    - fastavro >=1.0.0
+    - python-snappy >=0.5.4
+    - charset-normalizer >=1.3.6
+    - psutil >=4.0.0
+    - scipy >=1.4.1
+    - requests >=2.27.1,<2.28
+    - networkx >=2.5.1,<2.6
+    - typing-extensions >=3.10.0.2,<3.11
```

Unless this package imports itself at build time these are not needed. BTW, if it does need we should open an issue upstream to avoid this.

```suggestion
    - setuptools
    - pip
```